src/main/java/com/turbo/controller/OrdersController.java

```diff
@@ -0,0 +1,34 @@
+package com.turbo.controller;
+
+import com.turbo.service.PartService;
+import com.turbo.service.ReasonService;
+import com.turbo.service.TurbochargerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class OrdersController {
+
+    private TurbochargerService turbochargerService;
+    private PartService partService;
+    private ReasonService reasonService;
+
+    @Autowired
+    public OrdersController(TurbochargerService turbochargerService, PartService partService, ReasonService reasonService) {
+        this.turbochargerService = turbochargerService;
+        this.partService = partService;
+        this.reasonService = reasonService;
+    }
+
+    @GetMapping("/neworder")
+    public String newOrder (Model model){
```

Comment:
Proponuję zmienić nazwę metody jak i mappingu na np. create
teraz link do tej akcji będzie taki: http://www.domena.pl/orders/neworder
dodatkowo, to chyba powinien być POST a nie GET?

Reply:
Zmienilismy nazwy zgodnie z sugestia. Dodatkowo w naszym przypadku GET jest potrzebny zeby wyswietlic formularz z danymi z bazy. POST przekazemy w nastepnym pull requescie.